adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureParser.scala

```diff
@@ -177,6 +185,7 @@ class IntervalListParser extends Serializable {
             Feature.newBuilder()
               .setContig(Contig.newBuilder().setContigName(fields(0)).build())
               .setStart(fields(1).toLong - 1)
+              // todo: IntervalList ranges are closed, is this correct?
```

Comment:
If IntervalList intervals are 1-based and closed, shouldn't the next line be
```scala
.setEnd(fields(2).toLong)
```
E.g. `[7-9]` in IntervalList should be `[6-9)` in our 0-based, closed open system.